File: Herencia/ejercicio 2/pinguino.py
from animal import Animal

class Pinguino(Animal):
    def __init__(self, nombre: str, edad: int):
        super()._init_(nombre, edad)

    def desplazarse(self):
        print(f"{self.nombre} nada y se desliza en el hielo.")

    def __str__(self):
        return f"Pinguino(nombre={self.nombre}, edad={self.edad})"

File: Herencia/ejercicio 2/Pinguino.java
package herencia;
public class Pinguino extends Animal {

    public Pinguino(String nombre, int edad) {
        super(nombre, edad);
    }

    @Override
    public void desplazarse() {
        System.out.println(nombre + " nada y se desliza en el hielo.");
    }
}